crates/aptos-crypto/src/traits.rs

```diff
@@ -155,20 +180,20 @@ pub fn signing_message<T: CryptoHash + Serialize>(message: &T) -> Vec<u8> {
 /// This convertibility requirement ensures the existence of a
 /// deterministic, canonical public key construction from a private key.
 pub trait PublicKey: Sized + Clone + Eq + Hash + ValidCryptoMaterial +
-    // This unsightly turbofish type parameter is the precise constraint
-    // needed to require that there exists an
-    //
-    // ```
-    // impl From<&MyPrivateKeyMaterial> for MyPublicKeyMaterial
-    // ```
-    //
-    // declaration, for any `MyPrivateKeyMaterial`, `MyPublicKeyMaterial`
-    // on which we register (respectively) `PublicKey` and `PrivateKey`
-    // implementations.
+// This unsightly turbofish type parameter is the precise constraint
+// needed to require that there exists an
+//
+// ```
+// impl From<&MyPrivateKeyMaterial> for MyPublicKeyMaterial
+// ```
+//
+// declaration, for any `MyPrivateKeyMaterial`, `MyPublicKeyMaterial`
+// on which we register (respectively) `PublicKey` and `PrivateKey`
+// implementations.
```

Comment:
why the tab changing? this doesn't look right

Reply:
Fixed it back, strange formatter didn't fix it.